nautobot/utilities/utils.py

```diff
@@ -187,6 +188,17 @@ def serialize_object_v2(obj):
     return data
 
 
+def model_to_dict(obj, dotted_path=None):
+    """
+    Convert a Django model instance to a dictionary that is JSON serializable, leveraging a Django serializer.
+    """
+    if dotted_path:
+        # By default serialize_object_v2 will find a serializer, this is used to send in the serializer
+        # you would prefer, via a `import_string` dotted path to the serializer
+        return json.loads(JSONRenderer().render(import_string(dotted_path)(obj, context={"request": None}).data))
+    return json.loads(JSONRenderer().render(serialize_object_v2(obj)))
```

Comment:
How does this differ from just directly calling `serialize_object_v2(obj)`?

Reply:
doh!!!

I got confused by not having standard JSON types, but in looking at it, it is just OrderedDict, which is better served with the standard `json.loads(json.dump(serialize_object_v2(obj))`